protobuf/ports/port_msg.proto

```diff
@@ -53,6 +53,11 @@ message UnixSocketPortArg {
   /// Use a negative number for unthrottled polling. If unspecified or 0,
   /// it is set to 50,000 (50 microseconds, or 20k polls per second)
   int64 min_rx_interval_ns = 2;
+
+  /// If set, the port driver will send a confirmation once
+  /// the port is connected.  This lets pybess avoid a race during
+  /// testing.  See bessctl/test_utils.py for details.
+  bool confirm_connect = 3;
```

Comment:
I'll make two notes here on my own commit: (1) perhaps we should just always confirm? (2) if not, would it be better to make the confirm-packet-contents a `bytes` here rather than a magic string? (I was originally going to use a zero-size packet...)